src/wallet/wallet.ts

```diff
@@ -1175,10 +1177,16 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
       throw new Error('Wallet not ready yet.');
     }
 
-    const sign = this.signMessage(privKey, timestamp, this.walletId);
-    const data = await walletApi.createAuthToken(this, timestamp, privKey.xpubkey, sign);
+    try {
+      const sign = this.signMessage(privKey, timestamp, this.walletId);
+      const data = await walletApi.createAuthToken(this, timestamp, privKey.xpubkey, sign);
```

Comment:
I don't get it... Is this a fix? Why are you try/catching it? Maybe add a log?

Reply:
Yes, it's a fix!
We were having errors without any relation to whatever was being tested because this function runs asynchronously and crashes when the original caller context had already been fully finished.

Added a console on https://github.com/HathorNetwork/hathor-wallet-lib/pull/949/commits/9b045be121bd3e6583691d01ffab6cc968e59307